docs/blog/posts/2023/sr-linux-kubernetes-anycast-lab.md

```diff
@@ -0,0 +1,955 @@
+
+
+### links
+And finally, let's see how we interconnect all elements:
+
+```yaml title="Defining links"
+topology:
+# -- snip --
+
+  links:
+  ### #### fabric ### ####
+    - endpoints: ["spine1:e1-1", "leaf1:e1-49"]
+    - endpoints: ["spine1:e1-2", "leaf2:e1-49"]
+    - endpoints: ["spine1:e1-3", "leaf3:e1-49"]
+    - endpoints: ["spine1:e1-4", "leaf4:e1-49"]
+    - endpoints: ["spine2:e1-1", "leaf1:e1-50"]
+    - endpoints: ["spine2:e1-2", "leaf2:e1-50"]
+    - endpoints: ["spine2:e1-3", "leaf3:e1-50"]
+    - endpoints: ["spine2:e1-4", "leaf4:e1-50"]
+
+    #### minikube ####
+    - endpoints: ["leaf1:e1-1", "cluster1:eth1"]
+    - endpoints: ["leaf2:e1-1", "cluster1-m02:eth1"]
+    - endpoints: ["leaf3:e1-1", "cluster1-m03:eth1"]
+
+
+    #### clients ####
+    - endpoints: ["cli1:eth1", "leaf1:e1-2"]
+    - endpoints: ["cli2:eth1", "leaf2:e1-2"]
+    - endpoints: ["cli3:eth1", "leaf3:e1-2"]
+    - endpoints: ["cli4:eth1", "leaf4:e1-2"]
+```
+This is the full definition of all the connections required. As you can see, we use the first port (`e1-1`) of every leaf switch to connect the k8s node and second port (`e1-2`) to connect a client for tests.
+
+
+!!! note
+    `eth0`minikube node interfaces are connected to the docker bridge of the host running the topology. `eth1` interfaces, connected to Leaf switches, will be the ones used by MetalLB to establish the BGP the sessions. As reachability is signaled through this interface, clients will also reach k8s cluster services though `eth1`.
+
+
+
+## Lab deployment
+
+
+First, clone the lab:
+
+```bash
+https://github.com/srl-labs/srl-k8s-anycast-lab.git && cd srl-k8s-anycast-lab
+```
+
+Open two different shell sessions. One will be used to deploy the Containerlab topology, the other to start the Minikube cluster:
+
+
+```bash title="shell #1"
+clab deploy --topo srl-k8s-lab.clab.yml
+```
+You will notice that Containerlab starts deploying switches and clients and waits for Minikube nodes to appear.
+
+In the second shell, start Minikube cluster:
+
+```bash title="shell #2"
+minikube start --nodes 3 -p cluster1
+```
+
+With this command, Minikube will start three k8s nodes (`cluster1`, `cluster1-m02` and `cluster1-m03`) under the profile `cluster`.
+
+At the same time, in shell1, you can see how Containerlab starts "patching" `eth1` cluster node interfaces to Leaf switches, as soon as they appear:
+
+```bash 
+INFO[0022] node "cluster1-m02" depends on external container "cluster1-m02", which is not running yet. Waited 22s. Retrying...
+INFO[0022] node "cluster1-m02" depends on external container "cluster1-m02", which is not running yet. Waited 22s. Retrying...
+INFO[0022] node "cluster1-m02" depends on external container "cluster1-m02", which is not running yet. Waited 22s. Retrying...
+INFO[0022] Creating virtual wire: metal-leaf2:e1-1 <--> cluster1-m02:eth1
```

Comment:
`metal-leaf2` is not the name used in the lab

Reply:
noted